ami/models/components/sioconv_multi_lr.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
```

Comment:
```suggestion
import math
```
ただlog演算が欲しいだけなら標準の `math`モジュールを使う方が健全です